src/Utils/MultiStepInput.ts

```diff
@@ -31,6 +32,7 @@ class InputFlowAction {
 
 type InputStep = (input: MultiStepInput) => Thenable<InputStep|void>;
 
+/* istanbul ignore next */
```

Comment:
Since interface is not calculated during coverage measurement, this is not needed